README.md

```diff
@@ -4,227 +4,178 @@
-
 This software is licensed under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0).
 
 See the LICENSE and NOTICE files that should have been provided along with this
 software for details.
 
-## Development environment set up
-
-To learn how to run the project once the environment has been set up, refer to
-the "Running" section.
-
-The first thing you will need is cloning the project:
-> `$ git clone git@github.com:3scale/apisonator.git`
+## Development
 
-Next cd into the directory, `cd apisonator`.
-
-### Containerized environment
-
-#### Prerequisites
-
-* Docker (requires version 1.10.0 or later)
+See the file [DEVELOPMENT](DEVELOPMENT.md)
 
-#### With Docker
+## Description
 
-This requires GNU Make and has a single step:
-
-1. Run: `make dev`
-
-This command will take care of downloading and building all dependencies. Once
-that is done, the process will be way faster the next time.
-
-The project's source code will be available in `~/apisonator` and sync'ed with
-your local apisonator directory, so you can edit files in your preferred
-environment and still be able to run whatever you need inside the Docker
-container.
-
-The listener service port (3000 by default) is automatically forwarded to the host machine. The `make dev` command can be executed with the environment variable `PORT` set to
-a different desired listener port value to forward on the host machine.
-
-This Docker container is persistent, so your changes will be kept the next time
-you enter it.
+This is Red Hat 3scale API Management Platform's Backend.
 
-Getting rid of the persistent container is done with `make dev-clean`, whereas
-removing its image is done using `make dev-clean-image`.
+It has the following components:
 
-Alternatively you can start a container with the service running with
-`make dev-service`. This rule is intended for when you want to test
-the service endpoint.
+### Apisonator listener
 
-#### Maintain your dependencies up-to-date
+It provides the point of entry of the API Management Platform's Backend. The Service management API (SM API) is provided to authorize and report consumer
+API requests.
 
-Changes in code sometimes translate into changes in dependencies. If that is the
-case, your container will lag behind in dependencies, and some things might just
-start breaking. You might want to make a habit of making sure dependencies are
-updated when you enter your container. Run this from the project directory:
+Three main operations can be performed with this API:
 
-> $ `bundle install`
+ * Report: Reports a number of hits to one or more metrics
+ * Auth: Authorize a request
+ * Authrep: Combination of both the above
 
-The container image has additional tools to handle these dependencies for
-multiple Ruby versions. Check out the `scripts` directory if you are curious.
+Make sure to read the corresponding documentation in [ActiveDocs](https://support.3scale.net/reference/active-docs) for
```

Comment:
the link is broken - we should ask for the correct link

Reply:
Yes, I was looking into that as I said on the initial comment of the PR :+1: I'll update the PR with the fix once I have the new link (if available)